buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -38,6 +37,8 @@ import org.gradle.api.tasks.javadoc.Javadoc
 import org.gradle.api.tasks.testing.Test
 import org.gradle.testing.jacoco.tasks.JacocoReport
 
+import java.util.concurrent.atomic.AtomicInteger
```

Comment:
nit: it looks like your IDE import order is incorrect for groovy files.

Reply:
probably `src/*/java/**` then, there can potentially be more source sets (e.g. I wonder if it would make sense to model integration tests as source sets and move them out of `test`, or model different flink versions as source sets instead of projects).

Reply:
fixed both, seems to work

Reply:
SGTM on the `src/*/java/**`.

Moving ITs out to a different source set would be one way to do it but this would move them out of the tests artifact that is generated and uploaded to maven and we would have to configure other projects that depend on these tests to possibly depend on both now.